rusqlite_migration/src/builder.rs

```diff
@@ -7,7 +7,7 @@ use crate::{loader::from_directory, MigrationHook, Result, M};
 /// Allows to build a `Vec<M<'u>>` with additional edits.
 #[derive(Default, Debug)]
 pub struct MigrationsBuilder<'u> {
-    migrations: Vec<M<'u>>,
+    migrations: Vec<Option<M<'u>>>,
```

Comment:
```suggestion
    // The option makes some methods like edit more ergonomic, but the expectation is that entering
    // and leaving any method, there will be no None in the Vec.
    migrations: Vec<Option<M<'u>>>,
```

I would add this comment about the underlying assumption, what do you think?